Jenkinsfile.groovy

```diff
@@ -281,7 +281,7 @@ node('helm && kubectl') {
                 helm upgrade --install --wait --namespace ${NAMESPACE} ${DEPLOYMENT} . \
                     -f values.yaml -f minimal-values.yaml -f test-versions.yaml \
                     --set api.app.debug=true \
-                    --set postgresql.postgresqlPassword=notaproductionpassword \
+                    --set global.postgresql.postgresqlPassword=notaproductionpassword \
```

Comment:
Why the `global`? Does it have a meaning or is it just a cosmetic change?

Reply:
Bit of both. In our case it doesn't (yet) make any difference but the `global` does have significance - in that when you have multiple sub-charts that want to access the same variable it needs to be declared as global. https://github.com/helm/helm/blob/master/docs/chart_template_guide/subcharts_and_globals.md#global-chart-values